### PR TITLE
Address CG alerts for UA

### PR DESCRIPTION
**Pull request by sunandabalu**

### CODE REVIEW

File: tests/tool/Integration.Tests/IntegrationScenarios/WebLibrary/csharp/Original/packages.config [hunk 1]
```diff
@@ -4,5 +4,5 @@
   <package id="Microsoft.AspNet.Razor" version="3.2.7" targetFramework="net472" />
   <package id="Microsoft.AspNet.WebPages" version="3.2.7" targetFramework="net472" />
   <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net472" />
-  <package id="Newtonsoft.Json" version="6.0.5" targetFramework="net472" />
+  <package id="Newtonsoft.Json" version="12.0.2" targetFramework="net472" />
```
Comment (mjrousos) [inline tests/tool/Integration.Tests/IntegrationScenarios/WebLibrary/csharp/Original/packages.config:7]: This is the "before" project for an end-to-end test. It's meant to have out-of-date packages that UA should update. What's the reason we need to change them?

**Review by mjrousos — CHANGES_REQUESTED** [review]

File: tests/tool/Integration.Tests/IntegrationScenarios/WebLibrary/csharp/Original/packages.config (hunk 1 above)
Comment (sunandabalu) [reply tests/tool/Integration.Tests/IntegrationScenarios/WebLibrary/csharp/Original/packages.config:7]: CG alerting on this, and we need to upgrade the package versions to address the alert.

Comment (mjrousos) [reply tests/tool/Integration.Tests/IntegrationScenarios/WebLibrary/csharp/Original/packages.config:7]: Can we upgrade to a lower version? Seeing that UA successfully replaces old versions of packages that don't work with .NET 6 with newer versions that do work is a nice feature of this test.

Comment (sunandabalu) [reply tests/tool/Integration.Tests/IntegrationScenarios/WebLibrary/csharp/Original/packages.config:7]: Done, picked an older version that is not listed in CG

**Review by mjrousos — APPROVED** [review]
Is there any way to mark files as test code or non-shipping or something so that CG ignores them? 

These look fine for now to get things unblocked, but I am concerned that we're removing valuable test cases. One of the things UA should do is replace NuGet packages that don't support .NET 6 (like Newtonsoft v6) with newer versions that do support it. It would be good to find a way to keep old references only in test code for this purpose.

**Pull request merged**